download_torrent:
  save_path: /opt/data/movies
  catalog: /opt/data/catalog
  download_time: 1800.0
  download_pause: 21600

